### PR TITLE
[Fix #131] Add cljr-version

### LINKED ISSUE #131: Add a version command
Full version information for both `clj-refactor` and `refactor-nrepl`, down to the commit.

This makes it easier to deal with support requests.

### PR DESCRIPTION
**Pull request by expez**
We're also checking that the client and server versions are in sync and
if not we emit a warning

### CODE REVIEW

File: clj-refactor.el [hunk 1]
```diff
@@ -149,7 +152,10 @@ with the middleware."
 
 (defvar cljr--nrepl-ops
   '("artifact-list" "artifact-versions" "clean-ns" "configure" "find-symbol"
-    "find-unbound" "hotload-dependency" "resolve-missing"))
+    "find-unbound" "hotload-dependency" "resolve-missing" "version"))
+
+(defvar cljr-version "1.0.5-SNAPSHOT"
```
Comment (benedekfazekas) [inline clj-refactor.el]: is this a duplicate?

Comment (expez) [inline clj-refactor.el]: yes, it duplicates the version in the header of `clj-refactor.el`

Comment (benedekfazekas) [inline clj-refactor.el]: so you need `(defvar clj-version "1.0.5-SNAPSHOT")` twice?! sry for being slow here...

Comment (expez) [inline clj-refactor.el]: haha, I'm the one being slow here.  I was so annoyed by this duplication, with the version information appearing both in a var and in the header.  I spent like 10 mins trying to find a way to use `eval-when-compile` to extract the info from the header and put it into the var.

After all that I didn't see the literal duplication when you pointed it out.  Fixed in https://github.com/clojure-emacs/clj-refactor.el/commit/b0f1abaae0ea80b4ebeb45ac2f3474e426e050de

Comment (benedekfazekas) [inline clj-refactor.el]: :) :+1:

**Pull request merged**